src/model/queryable.ts

```diff
@@ -25,6 +25,12 @@ export interface BaseQueryOptions {
    * that are referenced from within the package.
    */
   externalPackages?: boolean;
+
+  /**
+   * Do not include any features that are only reachable via paths that include
+   * lazy import edges.
+   */
+  strictImports?: boolean;
```

Comment:
I'd been going with the idea that everything was default false, to match the semantics of `options.foo` without explicit defaults handling stuff.